src/visualizations/config/adapters/dhis_highcharts/pane/gauge.js

```diff
@@ -1,10 +1,9 @@
-const DEFAULT_PANE_SIZE = '140%'
-const DASHBOARD_PANE_SIZE = '100%'
+const DEFAULT_PANE_SIZE = '100%'
 
-export default function(dashboard) {
```

Comment:
To reflect the removed param this should be updated as well: https://github.com/dhis2/analytics/blob/a1e3fe5cdedd0e4f4d73528f7c5b461b033fb6a9/src/visualizations/config/adapters/dhis_highcharts/pane/index.js#L7